zoom_monitor.py

```diff
@@ -0,0 +1,108 @@
+from contextlib import contextmanager
+import time
+import urllib.parse
+
+from selenium.common.exceptions import (ElementClickInterceptedException,
+                                        NoSuchElementException)
+from selenium.webdriver import Chrome
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+
+
+@contextmanager
+def monitor_zoom(url):
+    zoom = ZoomMonitor(url)
+    try:
+        yield zoom
+    finally:
+        zoom.clean_up()
+
+
+class ZoomMonitor():
+    def __init__(self, url):
+        self._driver = Chrome()
+
+        updated_url = self._get_updated_url(url)
```

Comment:
Hm... maybe I merged too early. I don't like this name, but can't think of a better one. I'll change it next PR, when it's clearer what it should be instead. The updated URL is the "raw" URL after stripping out Google Calendar redirects and Zoom prompting you to use their app.